### DISCUSSION TITLE
Translation into Chinese of an English document.(4) 英文翻译为中文4

### DISCUSSION THREAD

**Discussion by OpenMMLab-Assistant-004** [open-mmlab/OpenMMLabCamp#498] [MMDetection]
# Translation into Chinese of an English document.(4) 
 英文翻译为中文4 

Translation [[https://github.com/open-mmlab/mmdetection/blob/dev-3.x/docs/en/user_guides/tracking_train_test.md](https://github.com/open-mmlab/mmdetection/blob/dev-3.x/docs/en/user_guides/tracking_train_test.md)](https://github.com/open-mmlab/mmdetection/blob/dev-3.x/docs/en/user_guides/tracking_train_test.md](https://github.com/open-mmlab/mmdetection/blob/dev-3.x/docs/en/user_guides/tracking_train_test.md))  into Chinese 

将 [[https://github.com/open-mmlab/mmdetection/blob/dev-3.x/docs/en/user_guides/tracking_train_test.md](https://github.com/open-mmlab/mmdetection/blob/dev-3.x/docs/en/user_guides/tracking_train_test.md)](https://github.com/open-mmlab/mmdetection/blob/dev-3.x/docs/en/user_guides/tracking_train_test.md](https://github.com/open-mmlab/mmdetection/blob/dev-3.x/docs/en/user_guides/tracking_train_test.md))  翻译为中文

## Task Information

**Repository**：MMDetection

**Difficulty**：Easy

**Points**：20

**Estimated Time Consumption**：21 days

## Task Status

**Claimant's Github ID**：zhangzhidaSunny

**Claim Time**：2023年07月25日

**PR Link**：https://github.com/open-mmlab/mmdetection/pull/10756

## Points to Note

❗️请注意：此任务属于超级视客营任务，不能被领取！请登陆 [超级视客营平台](https://openmmlab.com/activity/codecamp) 按照流程申请任务，我们会尽快审核～

**Comment by yeungkong** [comment]
Claim the mission

**Comment by zhangzhidaSunny** [comment]
https://github.com/open-mmlab/mmdetection/pull/10756
